frontends/web/src/i18n/en.json

```diff
@@ -648,6 +648,12 @@
                 "title": "How to securely pair with your phone",
                 "text": [ "After having downloaded our mobile app for either iOS or Android, you scan the displayed QR code, which sets up a secure channel between the mobile app and this application. Once scanned, follow the instructions in the mobile app." ]
             },
+            {   "title": "How does 2FA work",
```

Comment:
Move it below "What is a hidden wallet?" (since the Secrets are above Pairing).

Formulate it as a question (FAQ-like). Suggestion: "How does Two Factor Authorization (2FA) work?".

Reply:
Actually, moved "What is a hidden wallet?" below "How can I disable the two factor authorization (2FA)?", so the whole list order is even more consistent imho:

- "How can I eject the BitBox?"
- "How can I eject the SD card?"
- "How to securely pair with your phone"
- "How does Two Factor Authorization (2FA) work?"
- "How can I disable the two factor authorization (2FA)?"
- "What is a hidden wallet?"

Otherwise, you'd have:

- "How to securely pair with your phone"
- "What is a hidden wallet?"
- "How does Two Factor Authorization (2FA) work?"
- "How can I disable the two factor authorization (2FA)?"

In other words, doesn't feel like "hidden wallet" belongs in between phone pairing and 2FA.

Re: title. Updated to your suggestion.

![image](https://user-images.githubusercontent.com/25405/44161774-8fb47980-a0be-11e8-9d7b-8b89f250fdae.png)